recipe/build.sh

```diff
@@ -19,5 +21,20 @@ export INSTALL_INCLUDE="${PREFIX}/include"
 export BLAS="-lopenblas"
 export LAPACK="-lopenblas"
 
+# (optional) write out various make variables for easier build debugging
+eval ${LIBRARY_SEARCH_VAR}="${PREFIX}/lib" make config 2>&1 | tee make_config.txt
+
 eval ${LIBRARY_SEARCH_VAR}="${PREFIX}/lib" make -j1
+
+# make install below fails to link libmetis unless it is copied to $PREFIX/lib
+cp $SRC_DIR/lib/libmetis$DYNAMIC_EXT $PREFIX/lib
+
 make install
+
+
+if [ "$(uname)" == "Darwin" ]; then
+
+  # change link to METIS for CHOLMOD to reflect install path
+  install_name_tool -change $SRC_DIR/lib/libmetis.dylib @rpath/libmetis.dylib $PREFIX/lib/libcholmod.dylib
```

Comment:
Why is this needed? Doesn't `conda-build` fix this anyways? If not, do we need to do something similar on Linux with `patchelf`?